server/models/Request.js

```diff
@@ -4,35 +4,45 @@ const { ObjectId } = Schema.Types;
 const required = true;
 
 const requestSchema = new Schema({
-  user_id: {
+  userId: {
     type: ObjectId,
     ref: "User",
     required,
   },
-  sitter_id: {
+  sitterId: {
     type: ObjectId,
     ref: "User",
     required,
   },
   start: {
     type: Date,
     required,
+    validate: {
+      validator: function (start) {
+        const startMidnight = new Date(start);
+        const currentMidnight = new Date();
+
+        startMidnight.setHours(0, 0, 0, 0);
+        currentMidnight.setHours(0, 0, 0, 0);
+        return startMidnight >= currentMidnight;
+      },
+      message: "'start' must be current day or future",
+    },
   },
   end: {
     type: Date,
     required,
+    validate: {
+      validator: function (end) {
+        return end > this.start;
+      },
+      message: "'end' time must come after 'start' time",
+    },
   },
-  accepted: {
-    type: Boolean,
-    default: false,
-  },
-  declined: {
-    type: Boolean,
-    default: false,
-  },
-  paid: {
-    type: Boolean,
-    default: false,
+  status: {
+    type: String,
+    enum: ["pending", "accepted", "declined", "paid"],
+    default: "pending",
```

Comment:
Looks solid 💯 

I forgot to mention this before, but you can also use more [string SchemaTypes](https://mongoosejs.com/docs/schematypes.html#string-validators) here such as `trim` or `lowercase` 👍🏼